app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/disk/AuthDiskSourceImpl.kt

```diff
@@ -89,6 +90,14 @@ class AuthDiskSourceImpl(
         migrateAccountTokens()
     }
 
+    override var authenticatorSyncSymmetricKey: ByteArray?
+        set(value) {
+            val asString = value?.let { value.toString(Charsets.ISO_8859_1) }
+            putEncryptedString(AUTHENTICATOR_SYNC_SYMMETRIC_KEY, asString)
+        }
+        get() = getEncryptedString(AUTHENTICATOR_SYNC_SYMMETRIC_KEY)
+            ?.toByteArray(Charsets.ISO_8859_1)
```

Comment:
So this fixed the byte array length issue?

Reply:
It did! And I have the test to prove it:

https://github.com/bitwarden/android/pull/3905/files#diff-c72f2b9fbd3da15f026fec2053368bc6eb8f5bf2a233df05f9077c8809f2c6b2R1124

Reply:
Sweet!